src/MarkdownEditor2022.csproj

```diff
@@ -1,157 +1,95 @@
-      <SubType>Designer</SubType>
-      <Generator>VsixManifestGenerator</Generator>
-      <LastGenOutput>source.extension.cs</LastGenOutput>
-    </None>
-    <Content Include="Margin\highlight.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism-dark.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism.js">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Resources\Icon.png">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
+    <Content Include="..\LICENSE" Link="Resources\LICENSE" IncludeInVSIX="true" />
+    <Content Include="Margin\highlight-dark.css" IncludeInVSIX="true" />
+    <Content Include="Margin\highlight.css" IncludeInVSIX="true" />
+    <Content Include="Margin\md-template.html" IncludeInVSIX="true" />
+    <Content Include="Margin\prism-dark.css" IncludeInVSIX="true" />
+    <Content Include="Margin\prism.css" IncludeInVSIX="true" />
+    <Content Include="Margin\prism.js" IncludeInVSIX="true" />
+    <Content Include="Resources\Icon.png" CopyToOutputDirectory="PreserveNewest" IncludeInVSIX="true" />
   </ItemGroup>
+
   <ItemGroup>
-    <VSCTCompile Include="VSCommandTable.vsct">
-      <ResourceName>Menus.ctmenu</ResourceName>
-      <Generator>VsctGenerator</Generator>
-      <LastGenOutput>VSCommandTable.cs</LastGenOutput>
-    </VSCTCompile>
+    <VSCTCompile Include="VSCommandTable.vsct" ResourceName="Menus.ctmenu" />
   </ItemGroup>
+
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.mshtml, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <EmbedInteropTypes>True</EmbedInteropTypes>
-    </Reference>
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
-    <Reference Include="System.Design" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xaml" />
-    <Reference Include="WindowsBase" />
   </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Community.VisualStudio.VSCT" Version="16.0.29.6" PrivateAssets="all" />
     <PackageReference Include="Community.VisualStudio.Toolkit.17" Version="17.0.369" ExcludeAssets="Runtime" />
     <PackageReference Include="Markdig" Version="0.26.0" />
     <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.0.5232" PrivateAssets="all" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="3.4.231" PrivateAssets="all" />
+    <PackageReference Include="Unofficial.Microsoft.mshtml" Version="7.0.3300" PrivateAssets="all" />
```

Comment:
📝 I'm not thrilled about this dependency, but I wasn't aware of a simpler alternative

Reply:
My github build script handles versioning. This is not needed

Reply:
I'm referring to Unofficial.Microsoft.mshtml here. Nerdbank.GitVersioning is 💯

Reply:
➡️ This change has been eliminated since there is no more conversion to SDK-style project